modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -18,17 +18,21 @@ def metadata
         'zipCode' => @data.dig('veteran', 'address', 'postal_code') || '00000',
         'source' => 'VA Platform Digital Forms',
         'docType' => @data['form_number'],
-        'businessLine' => 'CMP'
+        'businessLine' => 'CMP',
+        'ssn_or_tin' => @data.dig('veteran', 'ssn_or_tin')
       }
     end
 
     def handle_attachments(file_path)
+      ssn_or_tin = @data.dig('veteran', 'ssn_or_tin')
```

Comment:
Since you added it to `metadata` can't you just call `metadata['ssn_or_tin']` instead of doing the @data dig again?